pkg/k8sutil/k8sutil.go

```diff
@@ -752,6 +761,12 @@ func (k *K8sutil) CreateClientMasterDeployment(deploymentType, baseImage string,
 			},
 		}
 
+		// disable secrets volume mount when ssl is not enabled
```

Comment:
@gianrubio sorry for the delay on this. I think the cleaner way is to not bring in the secret volume if we don't need unless you think that muddy's the code. So when the statefulsets / deployments are created, we can decide to add the secret volume or not. 

We'll need to add this check to the piece where certs are auto-generated since that's been recently added.